gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/GqlQuery.java

```diff
@@ -142,15 +138,14 @@ public boolean equals(Object obj) {
 
     @Override
     protected Object fromPb(byte[] bytesPb) throws InvalidProtocolBufferException {
-      return fromPb(DatastoreV1.GqlQueryArg.parseFrom(bytesPb));
+      return fromPb(com.google.datastore.v1beta3.GqlQueryParameter.parseFrom(bytesPb));
     }
 
-    static Binding fromPb(DatastoreV1.GqlQueryArg argPb) {
-      String name = argPb.hasName() ? argPb.getName() : null;
+    static Binding fromPb(com.google.datastore.v1beta3.GqlQueryParameter argPb) {
       if (argPb.hasCursor()) {
-        return new Binding(name, new Cursor(argPb.getCursor()));
+        return new Binding(null, new Cursor(argPb.getCursor()));
```

Comment:
I meant to ask about this; a GqlQueryParameter no longer knows it's name, so I'm not quite sure how to populate "name".  The query now stores the name as the key in the map "namedBindings," but I don't see a way to get that information here.